src/AppBundle/Controller/RepositoryController.php

```diff
@@ -40,13 +41,49 @@ public function indexAction(Request $request, Repository $repository)
             }
         }
 
+        $user = $this->getUser();
+        $repositoryStar = $this->get('doctrine')->getRepository('AppBundle:RepositoryStar')->findOneBy([
```

Comment:
Should be moved in a manager, as this code is also used in `starAction`, and could be used in the twig function `is_starred()`